### DISCUSSION TITLE
UTF-8 for export templates

### DISCUSSION THREAD

**Discussion by Brugal87** [nautobot/nautobot#6335] [Q&A]
I made a custom export template for our circuits, but some of the providers have "ñ" and "ó" in the name. When I export these circuits the names are like "TelefÃ³nica EspaÃ±a" instead of "Telefónica España". I tried to indicate the MIME type "text/plain; charset=utf-8" but it doesn't work. Any sugestion?

**Comment by glennmatthews** [comment]
Thanks for the report. This may be a bug. Can you check the encoding of the downloaded file (`file --mime my_export.txt`) to see whether it's an issue with the generated file encoding or an issue with the actual template rendering?
